include/abb_librws/rws_resource.h

```diff
@@ -26,6 +34,22 @@ namespace abb :: rws
      * \brief IO signal name.
      */
     std::string name;
+
+    bool equals(const SubscribableResource& rhs) const override
```

Comment:
I think you can compare resources by just comparing their URLs.

Reply:
That's what I thought of doing, but it requires more changes, because getting the URI is still only supported through 'SubscriptionManager'

Reply:
I think we should move resource URI calculation to resource classes.